tests/github_handler_test.rs

```diff
@@ -885,7 +886,7 @@ fn test_pull_request_merged_backport_labels() {
         slack::req("@joe.reviewer", msg, attach.clone()),
     ]);
 
-    let expect_thread = test.expect_will_merge_branches(vec!["release/1.0".into(), "release/2.0".into()]);
+    let expect_thread = test.expect_will_merge_branches(vec!["release/1.0".into(), "release/2.0".into(), "release/other-thing".into()]);
```

Comment:
I think this is really what #140 was talking about... I think folks wanted to use release-other-thing...

Reply:
but not restricting to `foo/bar` seems like a good idea anyway

Reply:
yeah. it was really only tangentially related. hence "cf." and not "fixes"... :sweat_smile: